cli/src/main/java/org/jboss/as/cli/gui/OperationDialog.java

```diff
@@ -471,23 +471,6 @@ private JComboBox makeJComboBox(List<ModelNode> values) {
             return new JComboBox(valueVector);
         }
 
-        // fill in form fields for write-attribute when an attribute node is selected.
-        private void setWriteAttributeValues() {
-            if (!OperationDialog.this.node.isLeaf()) return;
-            if (!OperationDialog.this.opName.equals("write-attribute")) return;
-
-            UserObject usrObj = (UserObject)OperationDialog.this.node.getUserObject();
-
-            if (this.name.equals("name")) {
-                ((JTextField)valueComponent).setText(usrObj.getName());
-                return;
-            }
-
-            if (usrObj.getValue().equals("undefined")) return;
-
-            if (this.name.equals("value")) ((JTextField)valueComponent).setText(usrObj.getValue());
-        }
```

Comment:
+1. Way back in 2012 the only call to this was deliberately changed to a call to a different method.